compiler/rustc_mir_build/src/thir/cx/expr.rs

```diff
@@ -278,14 +267,11 @@ impl<'tcx> Cx<'tcx> {
                 // When we apply adjustments to the receiver, use the span of
                 // the overall method call for better diagnostics. args[0]
                 // is guaranteed to exist, since a method call always has a receiver.
```

Comment:
this comment is now outdated and can be removed

Reply:
I don't think I've ever made a change without accidentally forgetting to delete the comment associated with such change.